gpi/frontend/web/src/ui/components/getty-tab-pane.vue

```
@@ -0,0 +1,43 @@
+<template>
+  <div
+    v-show="active"
+    class="pi-tab-pane">
+    <slot/>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'GettyTabPane',
+  props: {
+    label: {
+      type: String,
+      required: true,
+    },
+    name: {
+      type: Number,
+      required: true,
+    },
+    icon: {
+      type: String,
+      default: '',
+    },
+  },
+  computed: {
+    active () {
+      return this.$parent.currentName === this.name
+    },
+  },
+  mounted () {
+    this.$parent.addPanes(this)
```

We should check if the parent component implements the `addPanes` method. If it does not, we should throw an error and so they know they're not using the pane correctly.